.storybook/storybook-data.js

```diff
@@ -14,12 +14,12 @@ module.exports = [
 		},
 	},
 	{
-		id: 'global--buttons',
-		kind: 'Global',
+		id: 'components-button--hover-button',
+		kind: 'Components',
 		name: 'Buttons',
-		story: 'Buttons',
+		story: 'Default Button Hover',
 		parameters: {
-			fileName: './stories/buttons.stories.js',
+			fileName: './assets/js/components/Button.stories.js',
```

Comment:
I'm exactly sure what should happen with the scenario definition here, but we can't simply replace it with one of the single stories since the previous story used for VRT had multiple cases in the one story and this is just one.

If anything, maybe this should just be removed here and re-added later when we update the new stories to support defining scenarios.

Reply:
👍